patches/chrome-android-expectations-lint-suppressions.xml.patch

```diff
@@ -1,10 +1,10 @@
++  <issue id="ExportedService" severity="ignore"/><issue id="DuplicateDefinition"><ignore regexp="chrome/android/java/res/values/brave_colors.xml"/></issue><issue id="StringFormatCount"><ignore regexp="sponsored_images_bottom_sheet_text"/><ignore regexp="notification_marketing"/><ignore regexp="renew_monthly_save"/><ignore regexp="wallet_swap_fees_notice"/><ignore regexp="processing_fee"/></issue><issue id="WrongConstant"><ignore regexp="ContentSettingsType.BRAVE_CONTENT_SETTINGS_TYPES_LIST"/></issue><issue id="UnusedResources"><ignore regexp="The resource `R.string.playlist_.*` appears to be unused"/><ignore regexp="The resource `R.string.new_tab_otr.*` appears to be unused"/><ignore regexp="The resource `R.dimen.md_incognito_ntp.*` appears to be unused"/><ignore regexp="The resource `R.layout.incognito.*` appears to be unused"/><ignore regexp="The resource `R.layout.logo_view_layout` appears to be unused"/><ignore regexp="The resource `R.style.*_BrowserUI_Switch_Incognito` appears to be unused"/><ignore regexp="The resource `R.style.CardTransparentForDark` appears to be unused"/><ignore regexp="The resource `R.dimen.toolbar_hairline_height` appears to be unused"/><ignore regexp="The resource `R.style.TextAppearance_IncognitoNewTabLearnMoreLinkModern` appears to be unused"/><ignore regexp="The resource `R.string.accessibility_new_incognito_tab_page` appears to be unused"/><ignore regexp="The resource `R.drawable.incognito_splash` appears to be unused"/></issue><issue id="UnsafeOptInUsageError"><ignore regexp='(markerClass = androidx.media3.common.util.UnstableApi.class)'/></issue>
```

Comment:
Not really crytical, but I prefer just checking resources id exsitance for unused resources like we do here for example https://github.com/brave/brave-core/blob/176c7b5123fde7594b3c4c7d65e9f068f960d255/browser/tab_ui/android/java/src/org/chromium/chrome/browser/tab_ui/BraveTabUiThemeUtils.java#L17
This patch becomes way too overblown by these.

Reply:
I agree! Happy to make the changes.
What's the right place for checking drawables and strings?

Reply:
Feedback addressed in commit 👉 [`02b56f2` (#25850)](https://github.com/brave/brave-core/pull/25850/commits/02b56f277aafc6dde2fd202bf48e33a13528a744)